Neos.ContentRepository/Tests/Behavior/Features/Security/EditNodePrivilegeBasedOnDimension.feature

```diff
@@ -11,7 +11,7 @@ Feature: Privilege to restrict editing of nodes for a single dimension only
           'Neos.ContentRepository:EditServiceNodes':
             matcher: 'isDescendantNodeOf("/sites/content-repository/service")'
 
-          # EditEverything is needed, to switch to a "WHITELIST MODE" - i.e. where everything must be allowed explicitely.
+          # EditEverything is needed, to switch to a "ALLOWLIST MODE" - i.e. where everything must be allowed explicitely.
```

Comment:
Note: "Allowlist" was argued against in the Flow PR, since it is not a correct english term. Open for suggestions.

Reply:
What about "EXPLICIT ALLOW MODE"? (corresponds to the second part of the comment as in the first change.)

Reply:
Guess that would work, and then we can probably even scrap off the explanatory second part after the dash (see also my comment above). Thanks!
```suggestion
          # EditEverything is needed, to switch to an "EXPLICIT ALLOW MODE"
```

Reply:
Makes sense IMO